### ISSUE TITLE
Training Time and training setting for LGTSM in paper

### ISSUE THREAD

**Issue by hank19960918** [amjltc295/Free-Form-Video-Inpainting#33]
Hi Authors,

Thanks for sharing your interesting work. I follow the way you mentioned in paper to train(pre-train the generator without the
TSMGAN loss until convergence, and then fine-tune with the TSMGAN). But I can not reproduce the result in the paper. It would be a great help if you can mention the weight of different losses (i.e. perceptual, style, reconstruction and adversarial) you used while training. Thanks for your helping. 

Best,
Hank

**Comment by amjltc295** [comment]
Weights should be something like

Pretraining:
reconstruction: 1
perceptual: 1
style: 10

Fine-tuning:
plus adversarial: 1 

See https://github.com/amjltc295/Free-Form-Video-Inpainting/issues/30
